src/components/Team/ProfilePage/index.tsx

```
@@ -6,13 +6,13 @@ import {
   useUpsertTeamRecruitmentProfileMutation,
 } from 'api/teamRecruitmentProfile/queries';
 import LoadingSpinner from 'components/feedback/LoadingSpinner';
+import SubmitConfirmModal from 'components/Team/components/SubmitConfirmModal';
 import SubPageHeader from 'components/ui/SubPageHeader';
 import { FormProvider, useForm } from 'react-hook-form';
```

ESLint

[error] `react-hook-form` import should occur before import of `api/teamRecruitmentProfile/queries`

(import/order)